infracost-usage-example.yml

```
@@ -890,6 +890,7 @@ resource_usage:
     monthly_log_data_ingestion_gb: 20            # Monthly log data ingested by the workspace in GB (only used for Pay-as-you-go workspaces).
     monthly_additional_log_data_retention_gb: 30 # Monthly additional GB of data retained past the free allowance.
     monthly_log_data_export_gb: 40               # Monthly data in GB exported from the workspace.
+    monthly_sentinel_data_ingestion_gb: 40       # Monthly data in GB exported from Sentinel
```

Let's clarify when this usage param can be used:
```suggestion
    monthly_sentinel_data_ingestion_gb: 40       # Monthly data in GB exported from Sentinel. Only applicable when Microsoft Sentinel is enabled.
```